docs/gamejam/global-2021.html

```diff
@@ -389,7 +295,74 @@ <h2>Gallery</h2>
                         </div>
             -->
 
-
+            <section class="hoc-aside-02 background-papaya">
+                <div class="aside-text">
+                    <h3 class="title display-font">
+                        Follow us on Social Media
+                    </h3>
+                    <p>
+                        Tag us in your creations using @msmakecode or the
+                        hashtag #makecode. And don't forget to follow us to find
+                        out about the newest MakeCode Arcade updates, games, and
+                        competitions!
+                    </p>
+                    <div class="social-buttons">
+                        <a
+                            href="https://www.instagram.com/msmakecode"
+                            target="_blank"
+                            rel="noreferrer"
+                            onClick="window.pxtTickEvent('hourofcode2021.social.instagram')"
+                        >
+                            <i class="icon instagram"></i>
+                        </a>
+                        <a
+                            href="https://www.youtube.com/microsoftmakecode"
+                            target="_blank"
+                            rel="noreferrer"
+                            onClick="window.pxtTickEvent('hourofcode2021.social.youtube')"
+                        >
+                            <i class="icon youtube"></i>
+                        </a>
+                        <a
+                            href="https://twitter.com/msmakecode"
+                            target="_blank"
+                            rel="noreferrer"
+                            onClick="window.pxtTickEvent('hourofcode2021.social.twitter')"
```

Comment:
These ticks point to hourofcode2021. Shouldn't this be gamejam?

Reply:
I see this is just moved code. Probably don't want to change now.

Reply:
yeah my thought was this is the same footer and this gamejam is associated with the hour of code, so it wouldn't be super meaningful to differentiate the social media ticks... if we really wanted to split them maybe we could use the page url?

Reply:
How many scrolls does it take to get to the submissions? Would we want to replace the original header image with a much thinner one so the top row of submissions are visible? We could say 
"Top submissions: 2021 Climate Crisis Game Jam" and then move the video and description below the submissions?

I'm ok with the current design to get it up there, but would love to see an evolution to the suggestion above.